cgi-bin/DW/Controller/Support/Index.pm

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Support::Index
+#
+# This controller is for the Support Index page.
+#
+# Authors:
+#      hotlevel4 <hotlevel4@hotmail.com>
+#
+# Copyright (c) 2016 by Dreamwidth Studios, LLC.
+#
+# This is based on code originally implemented on LiveJournal.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Support::Index;
+
+use strict;
+use warnings;
+
+use DW::Controller;
+use DW::Routing;
+use DW::Template;
+
+DW::Routing->register_string( '/support/', \&index_handler, app => 1 );
+DW::Routing->register_string( '/support/index', \&index_handler, app => 1 );
+
+sub index_handler {
+    my $r = DW::Request->get;
+
+    my ( $ok, $rv ) = controller( anonymous => 1, form_auth => 1 );
```

Comment:
I don't see anywhere on this page where we are processing user input, so the `form_auth` argument should be removed.